src/services/resultsService.ts

```diff
@@ -87,8 +87,8 @@ export class ResultsService {
         const storedResults = ResultsService.get();
         const updatedResults = [
             ...storedResults.filter((result) => {
-                if (isWindows() && isPipInstall()) {
```

Comment:
Did you remove isPipInstall because it is not relevant for the check?

Reply:
I changed the sidebar update logic altogether for Windows by using the new method `formatWindowsAbsoluteFilePath`, so there's no difference between python and docker anymore.